srcpkgs/sane-airscan/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'sane-airscan'
+pkgname=sane-airscan
+version=0.99.24
+revision=1
+wrksrc="sane-airscan-${version}"
+build_style=gnu-makefile
+make_install_args="STRIP=-s"
```

Comment:
```suggestion
make_install_args="STRIP="
```

Actually, this would be best.

Reply:
Because they do allow us to override their stripping, which I like :)

Reply:
This completely disables stripping by the *make*. Does `xbps-src` strips the binaries elsewhere?

Reply:
btw I tried to do many variations of

```sh
make_install_args="STRIP=\"-s --strip-program=$STRIP\""
```

but I couldn't get around the quote expansion and the `make` inside `do_install()` always ends up being evaluated as if it was written like

```sh
make ... install "STRIP=-s" "--strip-program=$STRIP"
```

which is very frustrating so I'm quite happy to completely avoid the drama by not doing it :upside_down_face:

Reply:
one thing I believe you can do is `make_install_args="STRIP=-s STRIP+=--strip..."`, if you ever need such functionality :)

Reply:
> This completely disables stripping by the make. Does xbps-src strips the binaries elsewhere?

And yes, this happens in `common/hooks/post-install/06-strip-and-debug-pkgs.sh`